src/resolve/resolved-resolv-conf.c

```diff
@@ -228,11 +234,27 @@ static int write_resolv_conf_contents(FILE *f, OrderedSet *dns, OrderedSet *doma
         return fflush_and_check(f);
 }
 
+static int write_stub_resolv_conf_contents(FILE *f, OrderedSet *dns, OrderedSet *domains) {
```

Comment:
hmm, I think the old write_resolv_conf_contents() function should be renamed a bit to make clear what it actually contains now. Something like "write_uplink_resolv_conf_contents()" or so. That way we have the "stub" and the "uplink" version of resolv.conf, which I think should be pretty clear in their purposes

Reply:
Ack.